config/locales/en.yml

```diff
@@ -96,8 +96,9 @@ en:
   user_mailer:
     welcome_email:
       welcome_title: 'Welcome to OSBB Application '
-      welcome_message: 'You have successfully signed up to OSBB Application, your username is: '
-      before_link_text: 'To visit WEB site, just follow this link: '
+      welcome_message: 'You have successfully signed up to OSBB Application. '
```

Comment:
Use slim "=<" "=>" for spaces

Reply:
Thanks, a I was roll back it like was before, because it's no need now.